### DISCUSSION TITLE
Thanks for your quick and diligent work!

### DISCUSSION THREAD

**Discussion by anders8** [google/model-viewer#4360] [General]
So I'm at 3.1.0, and just did an update, and then got some errors related to the post-install script.... 

I come here to check, and whoops! I didn't realize that this release was only two hours old!

And then while I'm poking around as to the cause, I see that there's a **new** version that's just **THREE** minutes old!

Dunno if it fixes my issue, I don't think it's propagated to my repo yet, but I just want to say...

DANG, THANKS EVERYONE!

Because I can imagine the "Crap... crap... craaaaap!!!" internal screams as folks are scrambling to fix the build. ❤ So I'm just hanging out, I'll check back in a few days. Y'all do your thing. 👍 I believe in you!

**Comment by elalish** [comment]
Thanks! It's possible you're having some trouble related to this: https://github.com/google/model-viewer/pull/4359

This is a quick fix to get modelviewer.dev functioning again. This script should be a no-op unless you're updating the three.js version, which is my job. It probably needs some love - let us know if any of it is actually not working.
